substrate/client/executor/polkavm/src/lib.rs

```diff
@@ -26,13 +26,14 @@ use sp_wasm_interface::{
 };
 
 #[repr(transparent)]
-pub struct InstancePre(polkavm::InstancePre<()>);
+pub struct InstancePre(polkavm::InstancePre<(), polkavm::CallError>);
 
 #[repr(transparent)]
-pub struct Instance(polkavm::Instance<()>);
+pub struct Instance(polkavm::Instance<(), polkavm::CallError>);
 
 impl WasmModule for InstancePre {
 	fn new_instance(&self) -> Result<Box<dyn WasmInstance>, Error> {
+		// FIXME: Does not return `sc_executor_common::error::Error`.
```

Comment:
I'm confused by this FIXME. It does return `sc_executor_common::error::Error`?

Reply:
I don't know why it is there to be totally honest. This must be some left-over from early version (perhaps). Good catch anyway. I'll remove it.